docs/standard/base-types/parsing-datetime.md

```diff
@@ -19,57 +16,81 @@ helpviewer_keywords:
+
+The following example uses the <xref:System.Globalization.DateTimeStyles> enumeration to specify that the current date and time information should not be added to the <xref:System.DateTime> for unspecified fields.  
+
+[!code-csharp-interactive[Parsing.DateAndTime#3](../../../samples/snippets/csharp/how-to/conversions/StringToDateTime.cs#3)]
+[!code-vb[Parsing.DateAndTime#3](../../../samples/snippets/visualbasic/how-to/conversions/Program.vb#3)]
+ 
+## ParseExact
+
+The <xref:System.DateTime.ParseExact%2A?displayProperty=nameWithType> method converts a string that conforms to a specified string pattern to a <xref:System.DateTime> object. When a string that is not of the form specified is passed to this method, a <xref:System.FormatException> is thrown. You can specify one of the standard date and time format specifiers or a limited combination of the custom format specifiers. Using the custom format specifiers, it is possible for you to construct a custom recognition string. For an explanation of the specifiers, see the topics on [standard date and time format strings](standard-date-and-time-format-strings.md) and [custom date and time format strings](custom-date-and-time-format-strings.md).  
+
+Each overload of the <xref:System.DateTime.ParseExact%2A> method also has an <xref:System.IFormatProvider> parameter that provides culture-specific information about the formatting of the string. This <xref:System.IFormatProvider> object is a <xref:System.Globalization.CultureInfo> object that represents a standard culture or a <xref:System.Globalization.DateTimeFormatInfo> object that is returned by the <xref:System.Globalization.CultureInfo.DateTimeFormat%2A?displayProperty=nameWithType> property. However, unlike the other date and time parsing functions, this method also supports an <xref:System.IFormatProvider> that defines a non-standard date and time format.  
+
+In the following example, the <xref:System.DateTime.ParseExact%2A?displayProperty=nameWithType> method is passed a string object to parse, followed by a format specifier, followed by a <xref:System.Globalization.CultureInfo> object. This <xref:System.DateTime.ParseExact%2A> method can only parse strings that follow the long date pattern in the `en-US` culture.  
+
+[!code-csharp-interactive[Parsing.DateAndTime#4](../../../samples/snippets/csharp/how-to/conversions/StringToDateTime.cs#4)]
+[!code-vb[Parsing.DateAndTime#4](../../../samples/snippets/visualbasic/how-to/conversions/Program.vb#4)]
+
 ## See Also  
- [Parsing Strings](../../../docs/standard/base-types/parsing-strings.md)  
- [Formatting Types](../../../docs/standard/base-types/formatting-types.md)  
- [Type Conversion in .NET](../../../docs/standard/base-types/type-conversion.md)
+ [Parsing Strings](parsing-strings.md)  
+ [Formatting Types](formatting-types.md)  
+ [Type Conversion in .NET](type-conversion.md)  
+ [Standard DateTime formats](standard-date-and-time-format-strings.md)  
```

Comment:
I think that I would change DateTime in the titles for both the standard and custom format strings to "date and time", since that's the actual title and the strings are used by DateTimeOffset as well as DateTime.